### ISSUE TITLE
Let it Glow 2: Cannot click main menu buttons

### ISSUE THREAD

**Issue by n0samu** [ruffle-rs/ruffle#9685]
### Describe the bug

Link to SWF: http://i.notdoppler.com/files/letitglow2.swf

In [Let it Glow 2](http://www.notdoppler.com/letitglow2.php), clicking any of the main menu buttons does nothing.

### Expected behavior

Clicking Play Game should start the game, clicking Credits should show the credits screen, etc.

### Affected platform

Desktop app

### Operating system

Windows 10

### Additional information

A few errors are shown in the terminal which may or may not be the source of the problem:
```
WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.display.Graphics.beginGradientFill()
WARN ruffle_core::stub: Encountered stub: AVM2 flash.utils.Dictionary constructor with weak keys
ERROR ruffle_core::avm2::events: Error dispatching event EventObject(EventObject { type: "addedToStage", class: flash.events::Event, ptr: 0x227d3112840 }) to handler FunctionObject(FunctionObject { ptr: 0x227d2d628e0 }) : RustError("Attempted to call \"\" with 1 arguments (more than 0 is prohibited)")
WARN ruffle_core::stub: Encountered stub: AVM2 flash.display.Bitmap.pixelSnapping setter
WARN ruffle_core::stub: Encountered stub: AVM2 flash.display.Graphics.beginBitmapFill()
```

**Comment by Dinnerbone** [comment]
"Attempted to call \"\" with 1 arguments (more than 0 is prohibited)" is XML, usually

**Comment by Lord-McSweeney** [comment]
In this case it is. #9647 progresses this. In this case I believe it is unimplemented `xml.@*`, which should be equivalent to `XML.attributes`, but #9647 does not implement that, so the game errors out on `ReferenceError: Error #1069: Property * not found on XML and there is no default value.`.

**Comment by n0samu** [comment]
Fixed by #9911 and #9912.
